docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Playwright Test Automation Guidelines
+authors: 
+- Jees Mathew 
+- Karen Farrell 
+- Mihajlo Stojanovski 
+- James Perry
+tags:
+    - Default
+    - Test
+---
+
+## Introduction
+
+This document outlines best practices to follow when writing Playwright test scripts.
+
+## Locating selectors
+
+When planning testing at the readying phase of a work item, locators must be considered.  This allows Software Engineers to plan any additional work needed in order to enable our best practices.
```

Comment:
Would prefer to use the term Engineers here as there are cases where Test Engineers can create selectors.